model_compression_toolkit/exporter/target_platform_export/keras/exporters/fakely_quant_keras_exporter.py

```diff
@@ -0,0 +1,124 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import keras.models
+import tensorflow as tf
+import tensorflow_model_optimization as tfmot
+from keras.engine.base_layer import Layer
+from keras.engine.input_layer import InputLayer
+from tensorflow_model_optimization.python.core.quantization.keras.default_8bit.default_8bit_quantize_configs import \
+    NoOpQuantizeConfig
+
+from model_compression_toolkit.core.common import Logger
+from model_compression_toolkit.exporter.fully_quantized.keras.builder.quantize_config_to_node import \
+    SUPPORTED_QUANTIZATION_CONFIG
+from model_compression_toolkit.exporter.fully_quantized.keras.quantize_configs.weights_quantize_config import \
+    WeightsQuantizeConfig
+from model_compression_toolkit.exporter.target_platform_export.keras.exporters.base_keras_exporter import BaseKerasExporter
+
+
+class FakelyQuantKerasExporter(BaseKerasExporter):
+    """
+    Exporter for fakely-quant Keras models.
+    """
+
+    def __init__(self, model:keras.models.Model):
+        super().__init__(model)
+
+    def export(self) -> keras.models.Model:
+        """
+        Convert fully-quantized Keras model to a fakely-quant export-ready model.
+
+        Returns:
+            Fake-quant Keras model ready to export.
+        """
+
+        def _unwrap_quantize_wrapper(layer:Layer):
+            """
+            Convert layer wrapped with QuantizeWrapper to the layer it wraps, so it's ready to export.
+
+            Args:
+                layer: Layer to unwrap.
+
+            Returns:
+                Layer after unwrapping.
+
+            """
+
+            if isinstance(layer, tfmot.quantization.keras.QuantizeWrapper):
+                # For now, we assume only weights are quantized for models to export. Activation support is in progress.
+                if isinstance(layer.layer, InputLayer):
+                    if not isinstance(layer.quantize_config, NoOpQuantizeConfig):
+                        Logger.error(
+                            f'Currently, FakelyQuantKerasExporter supports weights quantization only. Please consider '
+                            f'disabling activation quantization, until supported ')
+                    return Layer() #Identity
+
+                if not isinstance(layer.quantize_config, tuple(SUPPORTED_QUANTIZATION_CONFIG)):
+                    Logger.error(
+                        f'Only supported quantization configs are {tuple(SUPPORTED_QUANTIZATION_CONFIG)}')
+
+                # If weights are quantized, use the quantized weight for the new built layer.
+                if isinstance(layer.quantize_config, WeightsQuantizeConfig):
+                    new_layer = layer.layer.__class__.from_config(layer.layer.get_config())
```

Comment:
this call looks a bit odd.. are you sure that is the correct way ti call "from_config" (using the __class__ attribute)?